engine/src/main/java/org/terasology/rendering/RenderHelper.java

```diff
@@ -72,25 +70,22 @@ public static float timeToTick(float time, float speed) {
     public static float evaluateOceanHeightAtPosition(Vector3f position, float days) {
         float height = 0.0f;
 
-        GLSLMaterial chunkMaterial = (GLSLMaterial) Assets.getMaterial("engine:prog.chunk").get();
-        ShaderParametersChunk chunkParameters = (ShaderParametersChunk) chunkMaterial.getShaderParameters();
-
-        float size = chunkParameters.waveSize;
-        float intens = chunkParameters.waveIntens;
-        float timeFactor = chunkParameters.waveSpeed;
+        float size = RefractiveReflectiveBlocksNode.waveSize;
+        float intens = RefractiveReflectiveBlocksNode.waveIntens;
```

Comment:
Please let's replace these "intens" strings with "intensity". Let's also rename size and intensity to waveSize and waveIntensity for the time being.  

Finally, let's add: 

`// TODO: rename to more meaningful/precise variable names, i.e. waveAmplitude or waveHeight.`